text/0092-replay-issue-creation.md

```diff
@@ -0,0 +1,55 @@
+- Start Date: 2023-05-15
+- RFC Type: decision
+- RFC PR: https://github.com/getsentry/rfcs/pull/92
+- RFC Status: draft
+
+# Summary
+
+We want to detect certain categories of issues only available through the Session Replay product. These issues can only be detected on the SDK. The Replay back-end will never have enough data to find these issues. For that reason this is primarily an SDK driven workload. The question is: what role should the Replay back-end have in Replay issue creation? Should the Replay SDK use the Replay back-end to generate new issues or should the SDK generate those issues through a generic, non-replay-specific interface?
```

Comment:
> We want to detect certain categories of issues only available through the Session Replay product.

Since we _can_ detected these categories of issues without the help of the Replay SDK all together, we should probably reword the "issues only available through the Session Replay product." language. How about, "issues that can greatly benefit from videos and context in the Session Replay product." or similar?